### PR TITLE
Create 2020-03-29-10lb-pork-shoulder.md

### PR DESCRIPTION
**Pull request by zmagg**

### CODE REVIEW

File: _posts/meat/2020-03-29-10lb-pork-shoulder.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Ingredients:
+* one 10 pound pork shoulder, or other pork roast cut
+* 1.5 cups of Diamond Crystal kosher salt
+* 0.5 cup granulated sugar
+
+Recipe:
+At large American grocery stores you can usually find a 10 pound pork shoulder in plastic packaging. Sometimes it's labeled "pork butt", "picnic", "shoulder", "blade roast". These are all subtly different parts of the pig, and cut differently, but rather than get into American animal husbandry practices, let's continue with the cooking, yah? 
+
+The day you bring the pork home, and at least 12 hours before cooking, salt the roast aggressively with ~1-1.15 cups of salt. Depending on how I'm feeling, I might also use half a cup of sugar. Let hang out in the fridge. You can either cover it in plastic wrap or just leave it open in a bowl. If you choose to wrap it,make sure to put it in a bowl as well, as it *will* leak liquids.
```
Comment (mookerji) [inline _posts/meat/2020-03-29-10lb-pork-shoulder.md]: nit: `may` not `might`. `may` conveys the possibility or option, but `might` expresses a hypothetical (generally conditional on something else).

File: _posts/meat/2020-03-29-10lb-pork-shoulder.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Day of cooking, remove all liquids and pat the roast dry. Place on a wire rack on a roasting pan (it'll leak fat as it cooks!), and roast for 6-8 hours at ~300 degrees fahrenheit. It's done when you can push a fork into it and twist easily.
```
Comment (mookerji) [inline _posts/meat/2020-03-29-10lb-pork-shoulder.md]: > *on the* day of cooking

File: _posts/meat/2020-03-29-10lb-pork-shoulder.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+The day you bring the pork home, and at least 12 hours before cooking, salt the roast aggressively with ~1-1.15 cups of salt. Depending on how I'm feeling, I might also use half a cup of sugar. Let hang out in the fridge. You can either cover it in plastic wrap or just leave it open in a bowl. If you choose to wrap it,make sure to put it in a bowl as well, as it *will* leak liquids.
+
+Day of cooking, remove all liquids and pat the roast dry. Place on a wire rack on a roasting pan (it'll leak fat as it cooks!), and roast for 6-8 hours at ~300 degrees fahrenheit. It's done when you can push a fork into it and twist easily. 
+
+OK, now what do you do with it? 
+
+You can serve the roast, as is, bo ssam style. Great for a dinner party. Read more here: https://www.epicurious.com/recipes/food/views/bo-ssam-51208610
+
+If you're cooking during quarantine, you might think, what can I do with a 10 pound roast and no friends? Here are some things I like to do:
```
Comment (mookerji) [inline _posts/meat/2020-03-29-10lb-pork-shoulder.md]: nit: `may`

File: _posts/meat/2020-03-29-10lb-pork-shoulder.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: Cook a 10 lb Pork Shoulder
```
Comment (mookerji) [inline _posts/meat/2020-03-29-10lb-pork-shoulder.md:3]: Possible alternative title: `Roast Pork Shoulder`, to be a food noun.

File: _posts/meat/2020-03-29-10lb-pork-shoulder.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+category: meat
+tags: recipeish
+---
+
+Ingredients:
+* one 10 pound pork shoulder, or other pork roast cut
```
Comment (mookerji) [inline _posts/meat/2020-03-29-10lb-pork-shoulder.md]: > 10 pound pork shoulder

File: _posts/meat/2020-03-29-10lb-pork-shoulder.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+Recipe:
+At large American grocery stores you can usually find a 10 pound pork shoulder in plastic packaging. Sometimes it's labeled "pork butt", "picnic", "shoulder", "blade roast". These are all subtly different parts of the pig, and cut differently, but rather than get into American animal husbandry practices, let's continue with the cooking, yah? 
+
+The day you bring the pork home, and at least 12 hours before cooking, salt the roast aggressively with ~1-1.15 cups of salt. Depending on how I'm feeling, I might also use half a cup of sugar. Let hang out in the fridge. You can either cover it in plastic wrap or just leave it open in a bowl. If you choose to wrap it,make sure to put it in a bowl as well, as it *will* leak liquids.
+
+Day of cooking, remove all liquids and pat the roast dry. Place on a wire rack on a roasting pan (it'll leak fat as it cooks!), and roast for 6-8 hours at ~300 degrees fahrenheit. It's done when you can push a fork into it and twist easily. 
+
+OK, now what do you do with it? 
+
+You can serve the roast, as is, bo ssam style. Great for a dinner party. Read more here: https://www.epicurious.com/recipes/food/views/bo-ssam-51208610
```
Comment (mookerji) [inline _posts/meat/2020-03-29-10lb-pork-shoulder.md]: This could be a Markdown link?

File: _posts/meat/2020-03-29-10lb-pork-shoulder.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+If you're cooking during quarantine, you might think, what can I do with a 10 pound roast and no friends? Here are some things I like to do:
+
+* Take out a portion, chop it up into smaller pieces, fat and solid meat, and fry (no need to add more fat) in a pan. After it's crispy, add some apple cider vinegar and some hot sauce and maybe some water. Let the liquids boil off, stirring the meat. Taste and add more salt, etc. Eat by itself, or on top of lettuce leaves or in tortillas.
```
Comment (mookerji) [inline _posts/meat/2020-03-29-10lb-pork-shoulder.md]: > Cut a portion, chop

or 

> chop up a smaller portion -- fat and solid meat --

File: _posts/meat/2020-03-29-10lb-pork-shoulder.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+OK, now what do you do with it? 
+
+You can serve the roast, as is, bo ssam style. Great for a dinner party. Read more here: https://www.epicurious.com/recipes/food/views/bo-ssam-51208610
+
+If you're cooking during quarantine, you might think, what can I do with a 10 pound roast and no friends? Here are some things I like to do:
+
+* Take out a portion, chop it up into smaller pieces, fat and solid meat, and fry (no need to add more fat) in a pan. After it's crispy, add some apple cider vinegar and some hot sauce and maybe some water. Let the liquids boil off, stirring the meat. Taste and add more salt, etc. Eat by itself, or on top of lettuce leaves or in tortillas. 
+* Take out a portion, like above. Chop up and let it fry and warm up in its own fat. Place at the bottom of a very large bowl. Chop up some lettuce/kale/chard/any form of greens. Boil some chinese noodles / ramen packet. Place noodles and hot water in bowl. Season with soy sauce, chili oil, chili crisp, etc. to taste. Soup noodles, yum.
```
Comment (mookerji) [inline _posts/meat/2020-03-29-10lb-pork-shoulder.md]: For the opening of this paragraph, see comment above.

File: _posts/meat/2020-03-29-10lb-pork-shoulder.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+* Use as the base for making congee.
```
Comment (mookerji) [inline _posts/meat/2020-03-29-10lb-pork-shoulder.md:30]: > Use as a congee base

or 

> Use as the base for congee.

File: _posts/meat/2020-03-29-10lb-pork-shoulder.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+You can serve the roast, as is, bo ssam style. Great for a dinner party. Read more here: https://www.epicurious.com/recipes/food/views/bo-ssam-51208610
+
+If you're cooking during quarantine, you might think, what can I do with a 10 pound roast and no friends? Here are some things I like to do:
+
+* Take out a portion, chop it up into smaller pieces, fat and solid meat, and fry (no need to add more fat) in a pan. After it's crispy, add some apple cider vinegar and some hot sauce and maybe some water. Let the liquids boil off, stirring the meat. Taste and add more salt, etc. Eat by itself, or on top of lettuce leaves or in tortillas. 
+* Take out a portion, like above. Chop up and let it fry and warm up in its own fat. Place at the bottom of a very large bowl. Chop up some lettuce/kale/chard/any form of greens. Boil some chinese noodles / ramen packet. Place noodles and hot water in bowl. Season with soy sauce, chili oil, chili crisp, etc. to taste. Soup noodles, yum.
+* Use as the base for making congee. 
+* Chop up and stir into stir fried vegetables.
+* You get the picture. It's endlessly versatile.
```
Comment (mookerji) [inline _posts/meat/2020-03-29-10lb-pork-shoulder.md:32]: Maybeeee: 


> * etc.
> 
> You get the picture! It's endlessly versatile.

File: _posts/meat/2020-03-29-10lb-pork-shoulder.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+tags: recipeish
+---
+
+Ingredients:
+* one 10 pound pork shoulder, or other pork roast cut
+* 1.5 cups of Diamond Crystal kosher salt
+* 0.5 cup granulated sugar
+
+Recipe:
+At large American grocery stores you can usually find a 10 pound pork shoulder in plastic packaging. Sometimes it's labeled "pork butt", "picnic", "shoulder", "blade roast". These are all subtly different parts of the pig, and cut differently, but rather than get into American animal husbandry practices, let's continue with the cooking, yah?
```
Comment (mookerji) [inline _posts/meat/2020-03-29-10lb-pork-shoulder.md]: > differently. But rather

File: _posts/meat/2020-03-29-10lb-pork-shoulder.md (hunk 1 above)
Comment (mookerji) [inline _posts/meat/2020-03-29-10lb-pork-shoulder.md]: > Let this hang out in the fridge

File: _posts/meat/2020-03-29-10lb-pork-shoulder.md (hunk 3 above)
Comment (mookerji) [inline _posts/meat/2020-03-29-10lb-pork-shoulder.md]: > you might think:

**Review by mookerji — CHANGES_REQUESTED** [review]
<3

File: _posts/meat/2020-03-29-10lb-pork-shoulder.md (hunk 4 above)
Comment (zmagg) [reply _posts/meat/2020-03-29-10lb-pork-shoulder.md:3]: ugh

Comment (zmagg) [reply _posts/meat/2020-03-29-10lb-pork-shoulder.md:3]: is this a blocking comment

**Comment by zmagg** [comment]
<3

**Pull request merged**